src/index.js

```diff
@@ -1,8 +1,21 @@
+// Append the node_modules of the github workspace and the node_modules of this action
+// to NODE_PATH. This supports lighthouse plugins - all the workspace needs to do it
```

Comment:
I think this comment is enough, then explain more in detail in the docs.

```suggestion
// to NODE_PATH.
```